Developers/Newcomers_Guide.rst

```diff
@@ -158,7 +158,7 @@ coala **clean** and **stable**.
 
```

Comment:
also, your shortlog is quite long, should be at maximum 50 chars, leave it to "Newcomers_Guide.rst: Fix broken link"